### ISSUE TITLE
Build error when 'Jest' option chosen during setup

### ISSUE THREAD

**Issue by elSteeze** [jvandemo/generator-angular2-library#143]
This error was generated after attempting to run my initial 'npm run build' after creating a library.

```
MC02L20JSFFT4:ui-common-testing data$ npm run build

> testing@0.1.0 build /Users/data/data-repositories/testing
> gulp build


module.js:471
    throw err;
    ^

Error: Cannot find module 'node-sass-tilde-importer'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/cbentson/trinet-repositories/ui-common-library-testing-version/tools/gulp/inline-resources.js:9:23)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ui-common-library-testing-version@0.1.0 build: `gulp build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ui-common-library-testing-version@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/cbentson/.npm/_logs/2017-06-29T16_56_45_928Z-debug.log

```

Steps to reproduce:
1. 'yo angular2-library' to generate library boiler-plate code
2. leave name, email, name of project, and repo all blank during setup
3. choose jest for testing
4. run 'npm run build' from root of project as soon as library finishes building

The initial build works just fine when I choose the 'karma-jasmine' option for testing

**Comment by izifortune** [comment]
Must have missed the dependency during rebase.
Sorry about this its in pr #144

**Comment by jvandemo** [comment]
PR has been merged and released as v11.0.2 👍
